src/import/import.rs

```diff
@@ -78,6 +78,12 @@ impl<Client: ImportClient> ImportJob<Client> {
                 "retry_count" => %retry_count,
                 "current_round" => %i,
             );
+            if retry_count > 0 && i == MAX_RETRY_TIMES - 1 {
```

Comment:
The `retry_count > 0` condition is unnecessary

Reply:
addressed